src/Symfony/Component/Cache/Tests/Adapter/PdoDbalAdapterTest.php

```diff
@@ -15,7 +15,6 @@
 use Doctrine\DBAL\Driver\AbstractMySQLDriver;
 use Doctrine\DBAL\DriverManager;
 use Doctrine\DBAL\Schema\Schema;
-use Doctrine\DBAL\Version;
```

Comment:
I know your fingers are itchy, but try to not do unrelated CS fixes. 

(You dont need to revert this. It was just an FYI)

Reply:
Sorry, my mistake with php-cs-fixer.